.github/workflows/tests.yaml

```diff
@@ -18,15 +18,24 @@ jobs:
     - name: Install backend version
       run: |
         tar -C /usr/local -xzf /root/r2inference/backends/tensorflow/v1.15.0/libtensorflow-cpu-linux-x86_64-1.15.0.tar.gz
-        ldconfig     
+        ldconfig
     - name: Configure the project
       run: |
         NOCONFIGURE=1 ./autogen.sh
         ./configure --disable-docs --enable-tensorflow
-    - name: Build
+    - name: Build library
       run: make
     - name: Run tests
       run: make check
+    - name: Install library
+      run: |
+        make install
+        ldconfig
```

Comment:
Is this necessary, or can we replace it by setting the env variable `LD_LIBRARY_PATH`?